static/sass/_notifications.scss

```diff
@@ -0,0 +1,5 @@
+.p-notification {
```

Comment:
According to the figma the is-dark variant should be correct, did a designer as for the color change?